include/mme-app/contextManager/s1HandoverProcedureContextManager.h

```diff
@@ -0,0 +1,63 @@
+ /*
+ * Copyright 2019-present, Infosys Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
```

Comment:
license needs fix.

Reply:
Taking care of License changes as part of next openmme prop activity. Will correct for all the files in one shot in the master branch.